cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -547,7 +547,6 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
                 effect=iam.Effect.ALLOW,
                 resources=[
                     self._format_arn(service="cloudformation", resource=f"stack/{Stack.of(self).stack_name}/*"),
-                    self._format_arn(service="cloudformation", resource=f"stack/{Stack.of(self).stack_name}-*/*"),
```

Comment:
I think this is needed with BYOS cause the plugin stack is deployed a substack and in the cookbook we need to describe it and retrieve the stack outputs.

Reply:
Sorry forgot to say that you cam safely remove it with batch and only leave it in the head node policy

Reply:
Removed for batch only, thanks